scipy/stats/_axis_nan_policy.py

```diff
@@ -403,12 +403,17 @@ def axis_nan_policy_wrapper(*args, _no_deco=False, **kwds):
                 # Note that *args can't be provided as a keyword argument
                 params = [f"arg{i}" for i in range(len(args))] + params[1:]
 
+            # raise if there are too many positional args
+            maxarg = (np.inf if inspect.getfullargspec(hypotest_fun_in).varargs
+                      else len(inspect.getfullargspec(hypotest_fun_in).args))
```

Comment:
Ok yes `varargs` is not None if we have something like `*param` and then `args` is an empty list.